extensions/wikia/VideosModule/Modules/VideosModuleStaff.class.php

```diff
@@ -38,7 +38,7 @@ public function getCacheKey() {
 		sort( $category );
 		$hashCategory = md5( json_encode( $category ) );
 
-		return implode( ':', $cacheKey, $hashCategory, $this->sort );
+		return implode( ':', [$cacheKey, $hashCategory, $this->sort] );
```

Comment:
@armonr this looks like a good catch that needed fixing, but does this address the original issue of empty results for the Videos Module? In the ticket it's the 'videos' array which is missing the videos, not the 'staffVideos'. Does this fix that as well?